draft-bishop-httpbis-http2-additional-certs.md

```diff
@@ -473,6 +489,10 @@ authentication request identifier, `Request-ID`. A peer that receives a
 error of type `PROTOCOL_ERROR`. Frames with identical request
 identifiers refer to the same `CERTIFICATE_REQUEST`.
 
+The `CERTIFICATE_NEEDED` frame MUST NOT be sent by servers prior to seeing the
+end of the client's request, and then MUST NOT be sent if the client included a
```

Comment:
Hmm, I just now realized that we are requiring that USE_CERTIFICATE be sent on a stream that is half-closed on the request side.  In fact, this would suggest that this is mandated.  That's going to be problematic in QUIC, where we don't have the option to send anything.